packages/x-date-pickers/src/locales/mk.ts

```diff
@@ -0,0 +1,79 @@
+import { PickersLocaleText } from './utils/pickersLocaleTextApi';
+import { getPickersLocalization } from './utils/getPickersLocalization';
+
+// This object is not Partial<PickersLocaleText> because it is the default values
+
+const mkPickers: PickersLocaleText<any> = {
+  // Calendar navigation
+  previousMonth: 'Предходен месец',
```

Comment:
```suggestion
  previousMonth: 'Претходен месец',
```

I stumbled on this PR and couldn't help but review :D @brsnik would you like to create a follow PR to fix this? :)

Reply:
@mnajdova Will do.